.github/workflows/test.yml

```
@@ -0,0 +1,223 @@
+
+    - name: Install Rust toolchain (stable for actual build)
+      uses: dtolnay/rust-toolchain@stable
+      id: stable-toolchain
+
+    - name: Set stable as default
+      run: rustup default ${{ steps.stable-toolchain.outputs.name }}
+
+    - name: Check minimal versions
+      run: |
+        cargo +nightly update -Z minimal-versions
+        cargo check --all-features
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)